superset/viz.py

```diff
@@ -395,6 +395,12 @@ def cache_key(self, query_obj, **extra):
         cache_dict["extra_cache_keys"] = self.datasource.get_extra_cache_keys(query_obj)
         cache_dict["rls"] = security_manager.get_rls_ids(self.datasource)
         cache_dict["changed_on"] = self.datasource.changed_on
+
+        url_params = self.form_data.get("url_params")
+
+        if url_params:
+            cache_dict["url_params"] = url_params
+
```

Comment:
I don't think this should be added here, as calling `url_params` in the Jinja context will now add the relevant values to `extra_keys`.

Reply:
This needs to be removed.